src/DynamoCoreWpf/UI/GuidedTour/GuidesManager.cs

```diff
@@ -31,8 +31,10 @@ public sealed class GuidesManager
 
         private DynamoViewModel dynamoViewModel;
 
-        private const double ExitTourVerticalOffset = 30;
-        private const double ExitTourHorizontalOffset = 0;
+        private RealTimeInfoWindow exitTourPopup;
+
+        private const double ExitTourVerticalOffset = 10;
+        private const double ExitTourHorizontalOffset = 110;
```

Comment:
Can you add some comments to how these two variables are used for future maintenance?

Reply:
I added some comments for those constants, see commit: https://github.com/DynamoDS/Dynamo/pull/11959/commits/c923ee41f84e324dd8d71b133e31219db5804ae7